docs/sources/tutorials/run-grafana-behind-a-proxy/index.md

```diff
@@ -123,6 +121,12 @@ server {
 }
 ```
 
+If your Grafana configuration does not set `serve_from_sub_path` to true then you need to add a rewrite rule to each location block:
+
+```
+ rewrite  ^/grafana/(.*)  /$1 break;
+```
+
 ## Configure HAProxy
 
 To configure HAProxy to serve Grafana under a _sub path_:
```

Comment:
We may also need to update those configs below, as they do something similar by rewriting the url.